package.json

```diff
@@ -51,8 +51,11 @@
     "karma-webpack": "^1.7.0",
     "legacy-loader": "0.0.2",
     "less": "^2.3.1",
+    "node-sass": "^3.4.2",
```

Comment:
Is this actually used somewhere? I figured ``sass-loader`` would have taken care of that dependency?